types/gulp-jasmine/index.d.ts

```diff
@@ -0,0 +1,52 @@
+// Type definitions for gulp-jasmine 2.4
+// Project: https://github.com/sindresorhus/gulp-jasmine#readme
+// Definitions by: Andrey Lalev <https://github.com/andypyrope>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+/// <reference types="node" />
+/// <reference types="jasmine" />
+
+declare namespace gulpJasmine { }
```

Comment:
If I'm right, the namespace can be removed.

Reply:
Thanks for mentioning this. Honestly, I thought it was a necessary workaround for creating a definition for a module-function but fortunately I was wrong.